src/frontend/src/lib/components/ui/Tag.svelte

```diff
@@ -30,5 +35,5 @@
 	class:py-0.5={size === 'sm' || size === 'md'}
 	class:md:py-1={size === 'sm' || size === 'md'}
 >
-	<slot />
+	{@render children?.()}
```

Comment:
Just checked, usually we do
```suggestion
	{#if nonNullish(children)}
		{@render children()}
	{/if}
```

Reply:
@daviddecentage TBH i think my implementation is cleaner. Also its the implementation that is recomended in the Migration Guide

Reply:
Ok then